contrib/fwupd.spec.in

```diff
@@ -96,6 +96,7 @@ Requires: bubblewrap
 Requires: shared-mime-info
 
 Recommends: python3
+Recommends: tpm2-tools tpm2-abrmd
```

Comment:
Likewise same recommends I think should apply to debian/ubuntu and arch.
Debian/Ubuntu name:
http://tracker.debian.org/pkg/tpm2-tools
https://tracker.debian.org/pkg/tpm2-abrmd

Arch name:
https://aur.archlinux.org/packages/tpm2-tools/

Reply:
https://aur.archlinux.org/packages/tpm2-abrmd